src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
Yeah I think I'll just merge this because all the other ways of preventing the issue are flawed too